manual/manual/tutorials/coreexamples.etex

```diff
@@ -61,9 +61,10 @@ usual basic data types: booleans, characters, and immutable character strings.
 "Hello world";;
 \end{caml_example}
 
-Predefined data structures include tuples, arrays, and lists. General
-mechanisms for defining your own data structures are also provided.
-They will be covered in more details later; for now, we concentrate on lists.
+Predefined data structures include tuples, arrays, and lists. There are also
+general mechanisms for defining your own data structures, such as records and
+variants, which
```

Comment:
Line-break shouldn't be here

Reply:
Is it really necessary to fill and justify the rest of the paragraph (the next 5 lines) in the tex *source* when it has no effect on the appearance of the output?  Seems to me it just makes more tedium for the editor and generates extraneous diffs in github.  What is your thinking on this?

Reply:
I'm not sure what the standard for the manual is (@Octachron), but this would certainly be yes for the main code base (e.g. for a re-worded documentation comment). While I completely agree that minimising diffs is important, so are coding standards and there are several places where I think the new lines also exceed the 80 char limit.

You're also assuming that the manual is only read after being TeX'd, but maybe I'm strange...

Reply:
How about if we go for whatever @Octachron suggests?

(Adhering strictly to an 80 character limit is, BTW, an additional manual step for me--tedious to inspect and correct.  There should be a tool to take care of this.  But let's not dive into this debate.)

Reply:
For now, I don't think that there is a strongly enforced character-limit, even if the manual tend to follow a soft-limit of 70-72  characters.

However, I agree that this line-break disrupts the reading flow, and the changes required for respecting the character limit are very moderate: 

> variants, which will be covered in more detail later; for now, we concentrate
> on lists. Lists are either given in extension as a bracketed list of

Note that for inspecting the 80 character limit, you can use the `check-typo` tools in the `tools`: `./tools/check-typo manual/manual/tutorials` show that there is three new lines that goes beyond the character limit (the other lines were already here before).

Reply:
I made all my changes less than 80 characters wide.  Hope we're good now.